README.md

```diff
@@ -119,6 +119,8 @@ the new testing approach:
 
 For checks that are not listed here, please refer to [Legacy development Setup](docs/dev/legacy.md).
 
+If you updated the test requirements for a check, you will need to run `tox --recreate` for changes to be effective.
```

Comment:
r/`you will need to run `tox --recreate` for changes t`/`run `tox --recreate` for changes t`